src/main/java/com/ait/lienzo/client/core/shape/wires/WiresLayoutContainer.java

```diff
@@ -401,4 +432,32 @@ public boolean equals(final Object o)
         }
     }
 
+    private static class ObjectAttribute
+    {
+        public Object obj;
+        public Attribute attr;
```

Comment:
just minor thing if you have time, if possible here, always better decoupling and immutability... so just `private final `:)